README.rst

```diff
@@ -0,0 +1,516 @@
+any warnings about ``tar: Ignoring unknown extended header keyword``.
+
+Go into the ``build_templates`` directory and copy over the closest
+``mkmf.template``._compiler.system\_ file into ``mkmf.template``.
+
+Edit it to set the NETCDF directory location if not in ``/usr/local`` or comment
+it out and set $NETCDF in your environment. *This NetCDF library must have been
+compiled with the same compiler that you use to compile DART and must include
+the F90 interfaces.*
+
+Go into ``models/lorenz_63/work`` and run *quickbuild.csh*.
+
+.. code-block::
+
+   $ cd models/lorenz_63/work
+   $ ./quickbuild.csh
+
+If it compiles, run this series of commands to do a very basic test:
+
+.. code-block::
+
+   $ ./perfect_model_obs
+   $ ./filter
+
+If that runs and you have Matlab installed on your system add
+``DART/diagnostics/matlab`` to your matlab search path and run the
+``plot_total_err`` diagnostic script while in the ``models/lorenz_63/work``
+directory. If the output plots and looks reasonable (error level stays around 2
+and doesn’t grow unbounded) you have successfully installed DART and completed
+your first assimilation with it.
+
+If you are planning to run one of the larger models and want to use the Lorenz
+63 model as a test, run ``./quickbuild.csh -mpi``. It will build filter and any
+other MPI-capable executables with MPI.
+
+.. important::
+
+   The ``mpif90`` command you use must have been built with the same version of
+   the compiler as you are using.
+
+If any of these steps fail or you don’t know how to do them, go to the DART
+project web page listed above for very detailed instructions that should get you
+over any bumps in the process.
+
+Citing DART
+-----------
+
+Cite DART using the following text:
+
+   The Data Assimilation Research Testbed (Version X.Y.Z) [Software]. (2019).
+   Boulder, Colorado: UCAR/NCAR/CISL/DAReS. http://doi.org/10.5065/D6WQ0202
+
+Update the DART version and year as appropriate.
+
+References
+----------
+
+.. [1] Anderson, J. L., T. Hoar, K. Raeder, H. Liu, N. Collins, R. Torn and A.
+       Arellano, 2009 The Data Assimilation Research Testbed: A Community 
+       Facility. *Bulletin of the American Meteorological Society*, **90**,
+       1283-1296, `doi:10.1175/2009BAMS2618.1
+       <http://dx.doi.org/10.1175/2009BAMS2618.1>`_
+
+.. |spaghetti_square| image:: ./guide/images/DARTspaghettiSquare.gif
+   :width: 100%
+
+.. |assim_anim| image:: ./guide/images/science_nuggets/AssimAnim.gif
+   :width: 100%
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Getting started
+
+   guide/system-requirements
+   guide/fortran-compiler
+   guide/locating-netcdf-library
+   guide/downloading-dart
+   guide/compiling-dart
+   guide/verifying-installation
+
+.. toctree::
+   :maxdepth: 2
+   :caption: What is data assimilation?
+
+   guide/introduction-ensemble-da
+   guide/lorenz-63-model
+   guide/da-in-dart-with-lorenz-63
+
+.. toctree::
+   :maxdepth: 2
+   :caption: What is DART?
```

Comment:
Is all this supposed to be here?
It shows up when you are on the GitHub code site.

Reply:
open question. The toctree directives will show up if you have the index for Sphinx and the GitHub readme as the same file

I would vote for having the GitHub as README.md.  For example, 
https://github.com/NCAR/wrf_hydro_py/blob/master/README.md
or
https://github.com/NCAR/VAPOR/tree/main

Reply:
Nothing we can do about it ... 
Could have content in README.md and toc stuff in README.rst ... TJH: I don't know which one actually gets shown on GitHub ...